File: C++/Courses/Course-2/Person.cpp
#include "Person.h"

Person::Person(){
    this->name = "Unknown!";
    this->surname = "Unknown!";
    this->age = 0;
}

Person::Person(string name, string surname, int age){
    this->age = age;
    this->name = name;
    this->surname = surname;
}

Person::Person(Person &obj){
    this->name = obj.name;
    this->surname = obj.surname;
    this->age = obj.age;
}

void Person::printInfo()
{
    cout << "Name: " << this->name << endl;
    cout << "Surname: " << this->surname << endl;
    cout << "Age: " << this->age << endl;
    cout << endl;
}

File: Java/Courses/Course-3/src/OOPI/Person.java
/**
 * Person
 */
package OOPI;

public class Person {
    // Attributes
    String name;
    int age;
    float salary;

    // Constructors
    
    Person(String name, int age, float salary){
        this.name = name;
        this.age = age;
        this.salary = salary;
    }

    Person(){
        this.name = "n/a";
        this.age = 0;
        this.salary = 0.0f;
    }

    Person(Person obj){
        this.name = obj.name;
        this.age = obj.age;
        this.salary = obj.salary;
        
    }


    void printInfo(){
        System.out.println("Name: " + name + "\nAge: " + age + "\nSalary: "+ salary +"\n");
    }
}